lib/game/game_widget.dart

```diff
@@ -94,11 +95,14 @@ class GameWidget<T extends Game> extends StatefulWidget {
 class _GameWidgetState extends State<GameWidget> {
   Set<String> activeOverlays = {};
 
+  Future<void> _gameLoaderFuture;
+  Future<void> get _memoizedGameLoaderFuture =>
```

Comment:
I don't think it needs the `memoized` part in the name?

Reply:
It sure don't, I just wanted a different name from the memoized variable, any suggestion?

Reply:
I thought it could just be named the same, but now when I look at the code I get why it can't.
Maybe a better name could be `_cachedGameLoaderFuture`? But also, I don't think it matters to much since it is a private field.

Reply:
IMO:
The getter: _gameLoaderFuture
The private field: _gameLoaderFutureCache

Reply:
Done